src/main/typescript/lib/formElements.tsx

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
please remove this file and all related dependencies for now. We'll gradually introduce this at a later moment. This file was just part of an experiment.